src/settings.py

```diff
@@ -83,7 +83,7 @@
 		'HOST': 'localhost',
 		'NAME': 'ppe_db',
 		# 'USER': 'hedgehog',
-		'PASSWORD': '_welc0me_',
+		'PASSWORD': 'welc0me',
```

Comment:
This must not be here, look up for python decouple, also there one specially for django if i remenber well, check it out